src/component/isolation.rs

```diff
@@ -209,6 +163,58 @@ impl ProcessIsolationController for DockerArchiveController {
 
 // TODO: Add a drop that gets rid of the loaded docker images after we're done with them
 
+#[derive(Debug)]
+pub struct ContainerizedScriptController {
+    executable_file: String,
+}
+
+impl ContainerizedScriptController {
```

Comment:
What is this? What is it's purpose?

Reply:
I think I explained this abstraction to you previously, maybe look at the rest of `isolation.rs` to jog your memory